### DISCUSSION TITLE
Stylex in Storybook "stylex.create should never be called. It should be compiled away."

### DISCUSSION THREAD

**Discussion by ajaxecho3** [facebook/stylex#292] [Q&A] [answered]
Hi! everyone 

May I trying to create a UI library base on Stylex  then now I'm encountering this error
![image](https://github.com/facebook/stylex/assets/42714378/182e3f7a-5dbe-4de6-83da-d8e1a8d01a75)
I just follow this installation instruction  https://stylexjs.com/docs/learn/installation/

**Comment by benekuehn** [comment]
Can you please share the source code to provide better help?

**Reply by ajaxecho3** [reply]
https://github.com/ajaxecho3/styljs-ui

**Comment by nmn** [comment, marked as answer]
Please follow the issue regarding Storybook integration.

**Reply by ajaxecho3** [reply]
Thank you :)

**Reply by akarpov91** [reply]
@nmn i'm also running into this. Can you provide the link to the Storybook integration issue?

**Reply by landonwjohnson** [reply]
nice answer!

**Comment by nmn** [comment]
Please try using the PostCSS plugin with Storybook. Along with a the Babel plugin for the babel config. That should work correctly and without issues in Storybook.
